### ISSUE TITLE
Unable to open an accordion with a screenreader

### ISSUE THREAD

**Issue by ddelrio1986** [react-bootstrap/react-bootstrap#3154]
When attempting to open an accordion item with a screen reader nothing happens. Here is an example:

https://codesandbox.io/s/wopo9y70yw

**Comment by suryagh** [comment]
@ddelrio1986 are you using keyboard to navigate with the sceenreader on? The sample you provided works for me on latest chrome

**Comment by ddelrio1986** [comment]
For keyboard it does work for me but when testing with VoiceOver if I use VO-left or VO-right to move to an accordion header the screen reader tells me that I am on a tab and that I can use VO-space to select the tab. Doing that says that I've selected the tab but it does not expand/contract.

**Comment by JustinSchieck** [comment]
Has anything come from this yet?

**Comment by Fettabachi** [comment]
You may need to click the enter key to get it to expand.

**Comment by ddelrio1986** [comment]
Hitting enter, space, and many other tested keys do not get it to open or close.
